examples/demo_threading/src/lib.rs

```diff
@@ -164,6 +176,9 @@ mod energy_usage {
                 qt_rx,
                 qt_tx,
                 join_handles: None,
+                sensors: Arc::new(Mutex::new(Arc::new(
```

Comment:
Why an Arc *inside* the Mutex instead of only outside?

Reply:
And if anything it's better than COW as if nothing is holding another Arc then it doesn't need to clone the data :tada: